### PR TITLE
Rabbitmq exporter env vars

### PR DESCRIPTION
**Pull request by bobo333**
#### What this PR does / why we need it:
The `rabbitmq-exporter` Dockerfile allows for additional environment variables to be passed in for configuration. This adds the missing environment variables to the helm chart so they may be configured. Specifically, it adds

- INCLUDE_VHOST
- SKIP_VHOST
- RABBIT_EXPORTERS
- OUTPUT_FORMAT
- RABBIT_TIMEOUT
- MAX_QUEUES

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by bobo333** [comment]
The linting step failed due to no maintainers being listed for the chart. I didn't remove them, they weren't there from the start. Is it ok to proceed despite this being the case?

**Comment by bobo333** [comment]
@mgoodness @unguiculus @seanknox anything I can do to help this one along?

**Comment by vishiy** [comment]
@bobo333 i see the same in my PR where by i clearly see maintainers. Did you resolve this ?

**Comment by bobo333** [comment]
@vishiy unfortunately I have not, I'm considering just putting myself as the maintainer

**Comment by bobo333** [comment]
I'm hoping one of the project owners can give some guidance on how to proceed

**Comment by unguiculus** [comment]
There's a type in `Chart.yaml`. That's why the linter fails

`mantainers`-> `maintainers`

**Comment by bobo333** [comment]
@unguiculus thank you for pointing that out, I probably should have found that myself.. 🤦‍♂️ 

regardless, looks like it's passing linting now, what are the next steps? thanks again!

**Comment by steven-sheehy** [comment]
Can you merge from master? I didn't see your PR and my [PR](https://github.com/helm/charts/pull/10273) had some overlap with yours.

**Comment by bobo333** [comment]
@steven-sheehy no problem, I've updated this PR with the current master branch, thanks!

File: stable/prometheus-rabbitmq-exporter/Chart.yaml
Comment (steven-sheehy) [inline stable/prometheus-rabbitmq-exporter/Chart.yaml]: Don't bump the appVersion. There is no v0.29.1 release: https://github.com/kbudde/rabbitmq_exporter/releases

File: stable/prometheus-rabbitmq-exporter/README.md
Comment (steven-sheehy) [inline stable/prometheus-rabbitmq-exporter/README.md]: Can you change this to ^$ to match values.yaml?

File: stable/prometheus-rabbitmq-exporter/README.md
Comment (steven-sheehy) [inline stable/prometheus-rabbitmq-exporter/README.md]: Remove extra period

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Comment by bobo333** [comment]
@steven-sheehy thank you for reviewing, I've addressed the comments

**Comment by steven-sheehy** [comment]
/ok-to-test
@unguiculus lgtm

**Comment by bobo333** [comment]
@steven-sheehy @unguiculus anything I can do to move this along? looks like it just needs a "lgtm" label and then can be merged?

File: stable/prometheus-rabbitmq-exporter/templates/deployment.yaml
Comment (unguiculus) [inline stable/prometheus-rabbitmq-exporter/templates/deployment.yaml:56]: In general, we want camel case, but since this matches the existing style, I'm going to accept this.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**
